Sources/TensorFlow/Operators/Math.swift

```diff
@@ -613,6 +613,19 @@ internal func _vjpFloor<T: TensorFlowFloatingPoint>(
     return (floor(x), { _ in Tensor(0).broadcasted(like: x) })
 }
 
+@inlinable
+@differentiable(vjp: _vjpSign(_:) where T: TensorFlowFloatingPoint)
+public func sign<T: TensorFlowNumeric>(_ x: Tensor<T>) -> Tensor<T> {
```

Comment:
Actually just `T: Numeric` is fine. The use site `Tensor<T>` implies the `T: TensorFlowScalar` constraint ;)